en/overrides.md

```diff
@@ -30,6 +30,46 @@ To remove all current overrides on a channel:
 
     sudo snap-proxy delete-override <snap> <channel>
 
+### Revisions and Architectures
+
+Every *.snap file has a unique
```

Comment:
Perhaps say "A  Snap Store channel can publish only one revision of a specific snap at any time". This phrasing avoids the implication that every snap is published on every channel.